app/Http/Controllers/FollowController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\User;
+use App\Models\Level;
+use Illuminate\Support\Facades\Auth;
+
+class FollowController extends Controller
+{
+    private $user;
+    private $level;
+
+    public function __construct(User $user, Level $level){
+        $this->user = $user;
+        $this->level = $level;
+    }
+
+
+    public function follower($id)
```

Comment:
here you can directly say `public function follower(User $user)` and you can remove `$user = $this->user->findOrFail($id);` same with the `following($id)` function.

Once implemented, you can totally remove `__construct` function and `private $user;` since we will no longer use that.